attention_lens/model/get_model.py

```diff
@@ -1,32 +1,60 @@
 import torch.types
 
 from typing import Union
-from transformer_lens import (
-    HookedTransformer,
-    HookedTransformerConfig,
-    FactoredMatrix,
-    ActivationCache,
-)
+from transformers import AutoModelForCausalLM, AutoConfig, AutoTokenizer
+# from transformer_lens import (
+#     HookedTransformer,
+#     HookedTransformerConfig,
+#     FactoredMatrix,
+#     ActivationCache,
+# )
 
 
-def get_model(
-    model_name: str = "gpt2-small", device: Union[str, torch.types.Device] = "cuda"
-) -> HookedTransformer:
-    """Loads and returns a pre-trained lens from the TransformerLens library by the given ``name``.
+# def get_model(
```

Comment:
Similarly here or elsewhere, lets not have dead code

Reply:
Resolved in latest commit.